README.md

```diff
@@ -1,2 +1,99 @@
-# tw-academy-s9
-Activity for session 9
+# **Sample Style Guide**            
+### Technical Writing Academy - May 2021
+
+### **Table of contents**
+
+#### [Bold](Bold)
+#### [Italics](Italics)
+#### [Fixed Font](Fixed-Font)
+#### [Abbreviations](Abbreviations)
+#### [Capitalization](Capitalization)
+#### [Acronyms](Acronyms)
+#### [Commas in Lists](Commas-in-Lists)
+#### [Numbers](Numbers)
+#### [Contractions](Contractions)
+#### [Apostrophes](Apostrophes)
+
+### **Bold**
+
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For
+example:
+* Click the **Create** button to start the process
```

Comment:
I am reviewing your document on my markdown application (I copied and pasted the text), and I see that you have to add a line break to apply the list format. Maybe it is just my application, but take a look at it.